extensions/wikia/PortableInfobox/PortableInfobox.i18n.php

```diff
@@ -3,9 +3,11 @@
 
 $messages[ 'en' ] = [
 	'portable-infobox-desc' => 'Create portable infoboxes which can be rendered using clean semantic HTML markup on
-	any skin / platform using using easy to understand powerful XML-like markup'
+	any skin / platform using using easy to understand powerful XML-like markup',
+	'unimplemented-infobox-tag' => 'Unimplemented infobox tag: <$1>'
 ];
 
 $messages[ 'qqq' ] = [
-	'portable-infobox-desc' => 'Portable Infobox extension description'
+	'portable-infobox-desc' => 'Portable Infobox extension description',
+	'unimplemented-infobox-tag' => 'Error message for using unimplemented infobox tag: <$1>'
```

Comment:
What do you think about phrasing it like: "Error message for using unimplemented infobox tag; $1 is the tag name without pointy braces"?